app.py

```diff
@@ -14,6 +14,7 @@
 # get the environment
 environment = utils.get_environment()
 stack_name_prefix = f"openchallenges-{environment}"
+stack_version = "0.0.11"
```

Comment:
`stack_version`?  isn't it the image or container version?

I think it would be better to set this [per environment in the cdk.json](https://github.com/Sage-Bionetworks-IT/openchallenges/blob/27d3e8571f8676cc6d80d4ff76c38dc27bbf3ebf/cdk.json#L66-L80)  because you will probably want to deploy different versions to different environments.  Getting VARS from cdk.json is below `# get VARS from cdk.json`

Reply:
All the images of the stack are now tagged with a shared non-moving tag (e.g. `0.0.11`). We can name this property `image_version` if you prefer.

If I want to deploy the images with the tag `0.0.11`, shouldn't we update the CDK app in the `dev` branch, then merge `dev` into `staging` and then `staging` into the `prod` branch as described in the read me?

Reply:
> We can name this property image_version if you prefer.

i think `image_version` sounds more appropriate.  

> shouldn't we update the CDK app in the dev branch, then merge dev into staging and then staging into the prod branch as described in the README? 

yes, good point.  I was thinking about cases where you might want to create another environment and control which version to deploy there.  Although I'm probably getting ahead of myself with this suggestion.  I guess it can be done as a follow on change.

Reply:
Done